.gitignore

```diff
@@ -20,3 +20,5 @@ docs/var/
 tools/ip_output.txt.back
 logs/*.log
 tools/logs/*.log
+modules/*/js/.module-cache/*
+htdocs//js/components/.module-cache/*
```

Comment:
There's a double slash here... and do you need the slash star at the end? Can't you just ignore the directory?